src/main/java/step1/domain/StringAddCalculator.java

```diff
@@ -0,0 +1,43 @@
+package step1.domain;
+
+import step1.exception.CharacterAndMinusCustomException;
+import step1.util.StringUtil;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private String value;
+    private static final Pattern COMPARE_NUMERIC = Pattern.compile("-?\\d+(\\.\\d+)?");
```

Comment:
필드 선언 순서는 상수다음 인스턴스 필드 순으로 오는게 컨벤션 입니다 :)
```suggestion
    private static final Pattern COMPARE_NUMERIC = Pattern.compile("-?\\d+(\\.\\d+)?");
    
    private String value;
```